sphinx/source/docs/user_guide/export.rst

```diff
@@ -105,22 +100,20 @@ to ``"svg"``.
     # option two
     plot.output_backend = "svg"
 
-Exporting an SVG Image
+Exporting an SVG image
```

Comment:
This whole section is very muddled (has been from the start). I think the "SVG Generation" headline should become "Exporting SVG images" and end with `export_pngs`. Then subsection for "limitations" and maybe finally a section about saving directly from browsers (e.g. with the bookmarlket)